src/Angor/Client/Pages/Recover.razor

```diff
@@ -662,6 +721,10 @@
             if (!response.Success)
                 return response;
 
+            var unconfirmedInfo = _cacheStorage.GetUnconfirmedInfo();
+            unconfirmedInfo.AddInputsAsPending(endOfProjectTransaction);
+            _cacheStorage.SetUnconfirmedInfo(unconfirmedInfo);
```

Comment:
If you have the same logic in a few methods in this file better have it as a private method so it's easier to understand what happens here.

Reply:
Good point